File: Unidad 3/Meta 3.1/Java/Biotopo.java
public class Biotopo {
	private String nombre;
	private Ecosistema ecosistema;

	public Biotopo(String nombre) {
		this.nombre = nombre;
	}

	public void setEcosistema(Ecosistema ecosistema) {
		this.ecosistema = ecosistema;
	}	

	public Ecosistema getEcosistema() {
		return ecosistema;
	}

	public String getNombre() {
		return nombre;
	}

	public void setNombre(String nombre) {
		this.nombre = nombre;
	}

	@Override
	public String toString() {
		return "Biotopo{" + "nombre=" + nombre + '}';
	}

}

File: Unidad 2/Meta 2.1/C#/Ecosistema/Biotopo.cs
using System;



public class Biotopo
{
    private string nombre;

    public Biotopo(string nombre)
    {
        this.nombre = nombre;
    }

    public string Nombre
    {
        get { return this.nombre; }
        set { this.nombre = value; }
    }

    override public string ToString()
    {
        return "Biotopo{" + "nombre=" + this.nombre + '}';
    }
}

